src/js/base/logic.js

```diff
@@ -268,3 +268,269 @@ ripe.Ripe.prototype.localeModel = function(
         fallback: fallback
     });
 };
+
+/**
+ * @ignore
+ */
+ripe.Ripe.prototype._toLocale = function(

```

Comment:
Delete. This already exists (look 10 lines above for the usage: `ripe.toLocale`)

Reply:
But the usage is not the same, this methods was copied (with some minor changes) from commons-pluginus and this seems to handle locale fallback unlike `toLocale` which handles more than that.

I added this method so that the `localeColor`, `localeMaterial`,` localePart` and `localeProperty` maintain their functionallity without messing around too much with the methods, maintaining consistency with this por to ripe-sdk